transfocate/tests/test_calculator.py

```diff
@@ -2,15 +2,17 @@
 # Standard #
 ############
 
+import numpy as np
 ###############
 # Third Party #
 ###############
```

Comment:
It seems like you've deleted these header import comments everywhere else, maybe they should be deleted here too.

Reply:
> It's picking pcdsdevices build 0 instead of build 2 and we fixed the lightpath dependency between the builds
> Though it's possible that the lightpath -> pydm -> pyqt chain would have failed if there wasn't a lightpath-dep-free build anyway
> I thought conda was supposed to always use the higher build number?

That was my understanding as well - I haven't a clue what the solver is doing here.